src/analytics.js

```diff
@@ -0,0 +1,11 @@
+/* global ga */
+
+(function(i,s,o,g,r,a,m){
+    i['GoogleAnalyticsObject'] = r;i[r] = i[r] || function(){
+        (i[r].q = i[r].q || []).push(arguments)
+    },i[r].l = 1 * new Date();a = s.createElement(o),
+m = s.getElementsByTagName(o)[0];a.async = 1;a.src = g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-72778945-2', 'auto');
+ga('send', 'pageview');
```

Comment:
What's changed about this file? It looks the same and could just be git being weird.